packages/react-input/Spec.md

```diff
@@ -61,3 +61,123 @@ Base accessibility information is included in the design document. After the spe
 - Identify UI parts that appear on **hover or focus** and specify keyboard and screen reader interaction with them
 - List cases when **focus** needs to be **trapped** in sections of the UI (for dialogs and popups or for hierarchical navigation)
 - List cases when **focus** needs to be **moved programatically** (if parts of the UI are appearing/disappearing or other cases)
+
+## Styling implementation notes
+
+Input has multiple size and appearance variants. These tables were created based on the design spec to assist with initial implementation and help ensure everything is handled. (They're not exactly part of the input spec, but this seemed like a reasonable place to put them.)
```

Comment:
I haven't decided. It was a huge pain initially pulling these from the design spec because a lot of values are specified in all appearances/sizes even though they never change. Even once pulled out, all the permutations of appearances, sizes, and states are complex enough that I wasn't sure it would be self-documenting. That might be less of an issue now that I've split up the styles per slot and per appearance (for example, if 2/4 appearances had the same background, previously I specified it in the "base" style; now I have it repeated in each appearance-specific style).

Reply:
I moved it to `Spec-styling.md` for now. I realized the spec PR itself is going to involve a bunch of "here are some notes on things v8/v0 do but we don't plan to support now, but might be relevant later" which should probably also go in different files to keep the main spec somewhat readable.